tests/errors.js

```diff
@@ -0,0 +1,139 @@
+import test from 'ava';
+import events from 'events';
+import httpMocks from 'node-mocks-http';
+import _ from 'lodash';
+
+import routes from '../lib/errors/routes';
+
+const EventEmitter = events.EventEmitter;
+
+// used in a promise below
+let nextCalled = false; // eslint-disable-line no-unused-vars
```

Comment:
Why are you setting this up outside the function call? Why can't this live inside the `next` mock?